route/build/src/main_route.f90

```diff
@@ -196,6 +219,16 @@ SUBROUTINE main_route(iens,           &  ! input: ensemble index
        RCHFLX_out(iens,ixRchProcessed(iSeg))%BASIN_QR(0) = RCHFLX_out(iens,iSeg)%BASIN_QR(1)   ! streamflow from previous step
        RCHFLX_out(iens,ixRchProcessed(iSeg))%BASIN_QR(1) = reachRunoff_local(iSeg)             ! streamflow (m3/s)
      end do
+
+     if (tracer) then
+       do iSeg = 1,nSeg
+         if (RCHFLX_out(iens,ixRchProcessed(iSeg))%BASIN_QR(1)>0) then
```

Comment:
Another "1" index.

Reply:
0 and 1 will be replaced with parameters. This change (issue #532) will be made in another PR.